lib/OctokitAction.ts

```diff
@@ -162,7 +163,47 @@ export abstract class OctokitAction extends Action {
     return result;
   }
 
+  protected async sendSlackMessage(text: string): Promise<void> {
+    const channel = this.getInput("slack-channel");
+    if (channel) {
+      this.log("Sending Slack message");
+      await this.sendSlackPost("https://slack.com/api/chat.postMessage", { channel, text });
+    } else {
+      this.log("Skip sending slack message, channel was not set.")
+    }
+  }
+
   private escapeRegex(string): string {
     return string.replace(/[/\-\\^$*+?.()|[\]{}]/g, '\\$&');
   }
+
+  private async sendSlackPost(url: string, jsonRequest: any): Promise<any> {
```

Comment:
I could have used `slack/web-api` npm package, but it has 3MB and 17 dependencies - that doesn't make sense to use.